src/coreclr/src/inc/crosscomp.h

```diff
@@ -113,12 +113,17 @@ typedef struct DECLSPEC_ALIGN(8) _T_CONTEXT {
 // each frame function.
 //
 
-#ifndef TARGET_UNIX
-#ifdef HOST_X86
-typedef struct _RUNTIME_FUNCTION {
+#if defined(HOST_WINDOWS)
+typedef struct _T_RUNTIME_FUNCTION {
     DWORD BeginAddress;
     DWORD UnwindData;
-} RUNTIME_FUNCTION, *PRUNTIME_FUNCTION;
+} T_RUNTIME_FUNCTION, *PT_RUNTIME_FUNCTION;
+#else
+#define T_RUNTIME_FUNCTION RUNTIME_FUNCTION
+#define PT_RUNTIME_FUNCTION PRUNTIME_FUNCTION
+#endif
```

Comment:
This is the inverse of the deleted code below except the `HOST_X86` term has been dropped, because I removed the definition  of `RUNTIME_FUNCTION` and `PRUNTIME_FUNCTION` which was in the `defined(HOST_WINDOWS) && defined(HOST_X86)` section below.